includes/migration-guide/retargeting/networking/default-value-servicepointmanagersecurityprotocol.md

```diff
@@ -3,7 +3,7 @@
-|Suggestion|This change affects applications that target the .NET Framework 4.7 or later versions.If you prefer to use a defined protocol rather than relying on the system default, you can explicitly set the value of the <xref:System.Net.ServicePointManager.SecurityProtocol?displayProperty=nameWithType> property.If this change is undesirable, you can opt out of it by adding a configuration setting to the [<runtime>](https://docs.microsoft.com/dotnet/framework/configure-apps/file-schema/runtime/runtime-element) section of your application configuration file. The following example shows both the <code>&lt;runtime&gt;</code> section and the <code>Switch.System.Net.DontEnableSystemDefaultTlsVersions</code> opt-out switch:<pre><code class="language-xml">&lt;runtime&gt;&#13;&#10;&lt;AppContextSwitchOverrides value=&quot;Switch.System.Net.DontEnableSystemDefaultTlsVersions=true&quot; /&gt;&#13;&#10;&lt;/runtime&gt;&#13;&#10;</code></pre>|
+|Suggestion|This change affects applications that target the .NET Framework 4.7 or later versions.If you prefer to use a defined protocol rather than relying on the system default, you can explicitly set the value of the <xref:System.Net.ServicePointManager.SecurityProtocol?displayProperty=nameWithType> property.If this change is undesirable, you can opt out of it by adding a configuration setting to the [<runtime>](~/docs/framework/configure-apps/file-schema/runtime/runtime-element.md) section of your application configuration file. The following example shows both the <code>&lt;runtime&gt;</code> section and the <code>Switch.System.Net.DontEnableSystemDefaultTlsVersions</code> opt-out switch:<pre><code class="language-xml">&lt;runtime&gt;&#13;&#10;&lt;AppContextSwitchOverrides value=&quot;Switch.System.Net.DontEnableSystemDefaultTlsVersions=true&quot; /&gt;&#13;&#10;&lt;/runtime&gt;&#13;&#10;</code></pre>|
```

Comment:
can you make the same change here? [\<runtime>] has again dropped out of the published topic.